capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
- [OPTIONAL] The requirement states that the `correct_name` method of the `CapitalizeDecorator` class should return the capitalized name of the `nameable`. The logic you wrote returns the upper case form of the nameable. To correct this, you need to use the `capitalize` method instead of the `upcase` method. 👍